### DISCUSSION TITLE
Prevent Panel Tab Event from Dragging to Editor.

### DISCUSSION THREAD

**Discussion by cemalgnlts** [mathuo/dockview#394] [Q&A] [answered]
Hi,

The title is not very descriptive but that is my question:
I have an editor inside the panel and when the panel is dragged, the drag event of the panel can go to the editor.

[Screen recording 2023-11-18 12.52.37.webm](https://github.com/mathuo/dockview/assets/45357531/03ea807b-1844-4b6f-a871-1195d1e544be)

**Comment by techbech** [comment, marked as answer]
This is really something you have to implement yourself. I'm not totally sure of how I would do it with the monaco editor, but the problem is that the monaco editor does accept dnd. A suggestion would be to overwritting the content of the default data transfer object and just leave the content empty. This is described here https://dockview.dev/docs/components/dockview#intercepting-drag-events. This would mean that just a empty string is inserted into the editor.

**Reply by techbech** [reply]
Or else one could maybe add support for disabling the propagation of dnd tabs on a single panel.

With that said, I'm actually using the tabs as dnd to a monaco editor where the tab represents a file, and dragging the tab into the editor the file path is inserted.

**Reply by cemalgnlts** [reply]
Thanks for the explanation and advice.
